how-to/how-to-template.md

```diff
@@ -0,0 +1,54 @@
+<!-- Copy this Template. -->
+<!-- Describe the title of your article by replacing "How To Template" with the page name you want to publish to. -->
+
+# Title
+
+::information_source: Before using this template, read the accompanying [how-to template guide](how-to-template-guide.md)
+
+## Overview
+
+This guide explains how to {insert a brief description of the task}.
+
+:information_source: Here are some examples:
+
+- This guide explains how to comment on a pull request.
+- This guide explains how to create or delete a branch on Github.
+- This guide explains how to create a repository.
+
+(Optional) Specify when and why your user might want to perform the task.
+
+## Before you begin
+
+<!-- Delete this section if your readers can go to the steps without requiring any prerequisite knowledge. -->
+
+Before you {insert a brief description of the task}, complete the following prerequisites:
+
+- Prerequisite 1
+- Prerequisite 2
+- Prerequisite 3
+
+## {Task name}
+
+1. {Write the step here. Use a verb to start.}
+
+:information_source: You can use this format to describe your step:
+
+Explanatory text
+
+(Optional) Code sample or screenshot that helps your users complete this step.
+
+(Optional) The result of completing this step.
```

Comment:
suggestion: An example will likely be very valuable here (or in the guide). Something like:
* Notice {button is visible at the bottom left of the display}
* Press {button}
* Observe {what has changed}
* Confirm {something has happened}